docs/root/intro/arch_overview/http/http_connection_management.rst

```diff
@@ -19,14 +19,14 @@ HTTP connection manager :ref:`configuration <config_http_conn_man>`.
 HTTP protocols
 --------------
 
-Envoy’s HTTP connection manager has native support for HTTP/1.1, WebSockets, and HTTP/2. It does not support
-SPDY. Envoy’s HTTP support was designed to first and foremost be an HTTP/2 multiplexing proxy.
+Envoy’s HTTP connection manager has native support for HTTP/1.1, WebSockets, HTTP/2 and HTTP/3. It does not support
+SPDY. Envoy’s HTTP support was designed to first and foremost be an HTTP/2+ multiplexing proxy.
```

Comment:
I think it's fine to keep this as "HTTP/2" since when Envoy was designed HTTP/3 was not yet a thing. Your call though.